canvas_sdk/v1/data/questionnaire.py

```diff
@@ -0,0 +1,189 @@
+from django.db import models
+
+from canvas_sdk.v1.data import Patient
+from canvas_sdk.v1.data.base import CommittableModelManager, ValueSetLookupQuerySet
+from canvas_sdk.v1.data.user import CanvasUser
+
+
+class ResponseOptionSet(models.Model):
+    """ResponseOptionSet."""
+
+    class Meta:
+        managed = False
+        app_label = "canvas_sdk"
+        db_table = "canvas_sdk_data_api_responseoptionset_001"
+
+    dbid = models.BigIntegerField(primary_key=True)
+    created = models.DateTimeField()
+    modified = models.DateTimeField()
+    status = models.CharField()
+    name = models.CharField()
+    code_system = models.CharField()
+    code = models.CharField()
+    type = models.CharField()
+    use_in_shx = models.BooleanField()
+
+
+class ResponseOption(models.Model):
+    """ResponseOption."""
+
+    class Meta:
+        managed = False
+        app_label = "canvas_sdk"
+        db_table = "canvas_sdk_data_api_responseoption_001"
+
+    dbid = models.BigIntegerField(primary_key=True)
+    created = models.DateTimeField()
+    modified = models.DateTimeField()
+    status = models.CharField()
+    name = models.CharField()
+    code = models.CharField()
+    code_description = models.CharField()
+    value = models.CharField()
+    response_option_set = models.ForeignKey(
+        ResponseOptionSet, on_delete=models.DO_NOTHING, related_name="options"
+    )
+    ordering = models.IntegerField()
+
+
+class Question(models.Model):
+    """Question."""
+
+    class Meta:
+        managed = False
+        app_label = "canvas_sdk"
+        db_table = "canvas_sdk_data_api_question_001"
+
+    id = models.UUIDField()
+    dbid = models.BigIntegerField(primary_key=True)
+    created = models.DateTimeField()
+    modified = models.DateTimeField()
+    status = models.CharField()
+    name = models.CharField()
+    response_option_set = models.ForeignKey(
+        ResponseOptionSet, on_delete=models.DO_NOTHING, related_name="questions"
+    )
+    acknowledge_only = models.BooleanField()
+    show_prologue = models.BooleanField()
+    code_system = models.CharField()
+    code = models.CharField()
+
+
+class Questionnaire(models.Model):
+    """Questionnaire."""
+
+    class Meta:
+        managed = False
+        app_label = "canvas_sdk"
+        db_table = "canvas_sdk_data_api_questionnaire_001"
+
+    objects = models.Manager.from_queryset(ValueSetLookupQuerySet)()
```

Comment:
`ValueSetLookupQuerySet` won't work on this model because there is no `codings` attribute.